README.md

```diff
@@ -155,15 +155,26 @@ Installation guidelines
 ### Easy method
 Install with `pip` by running `pip install hickle` from the command line.
 
+#### Install on Windows 32 bit
+
+Prebuilt Python wheels packages are available on PyPi until H5PY version 2.10 and Python 3.8.
+Any newer versions have to be built and installed Manually.
+
+1) Install h5py 2.10 with `pip` by running `pip install "h5py<3.0"` from the commandline
```

Comment:
I think you mean `pip install h5py==2.10`.

Reply:
But it isn't the same. If someone already has `h5py` installed, but it is not 2.10, your version won't update it, while mine will.

Reply:
Short h5py versions < 2.10 already installed is a case to be considered.and covered here. -> Will change